spec/controllers/follows_controller_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe FollowsController, type: :controller do
+  sign_in_user
+
+  let(:other_user) { create(:user) }
+  let(:question) { create(:question, user: @user) }
+  let(:follow) { create(:follow, user: @user, followable_id: question.id) }
+  let(:other_follow) { create(:follow, user: other_user, followable_id: question.id) }
+
+  describe 'POST #create' do
+    context 'correct work' do
```

Comment:
Where is a spec for "incorrrect work"? You should check, that a user cannot create two follows for the same question.